_topic_maps/_topic_map_osd.yml

```diff
@@ -1716,14 +1716,6 @@ Topics:
   Topics:
   - Name: Backup and restore by using VM snapshots
     File: virt-backup-restore-snapshots
-  - Name: Installing and configuring OADP
-    File: virt-installing-configuring-oadp
-  - Name: Backing up and restoring virtual machines
-    File: virt-backup-restore-overview
-  - Name: Backing up virtual machines
-    File: virt-backing-up-vms
-  - Name: Restoring virtual machines
-    File: virt-restoring-vms
```

Comment:
See my comment in the 4.16 PR. The topic map here should be the same as in `main` to avoid merge conflicts https://github.com/openshift/openshift-docs/pull/75004/files#diff-54b6c45a07b510755a4b365a48b7130a0369666209fd2e835b543de5e909e2f6R1658-R1659